### DISCUSSION TITLE
Is there away that can be used without the color part?

### DISCUSSION THREAD

**Discussion by michaelcrossland1979** [basnijholt/adaptive-lighting#606] [Q&A] [answered]
I would like to be able to just use this as a better way to slowly step down the lights instead of doing the way i have been with google. Where every so many hours after sun set it drops the lights by fixed amount till it's done but it's very jarring to say the least going from say 75% light to 50% is ver jarring if your not ready for it to happen. I would like to do like what amazon does where you set it to whatever light level you want it to come on at and it can ramp up the light to that at a set rate or it can slowly drop it down at a rate over that time frame. I don't much care for the fixed light color I want to just be able to use the light paction that I have already setup and just have it slow fade tyhe light levels over time that it not change the color  of my lights. I get why that is done and that great but not for everyone. If I had a way to I would most def send you some money for the work as a thank you and to help keep you working on this.

**Comment by bouwew** [comment, marked as answer]
You should be able to do this by turning off the relevant color-switches and keeping the brightness-switches to on.
